cmd/cloud/installation.go

```diff
@@ -68,6 +68,10 @@ var installationCreateCmd = &cobra.Command{
 		database, _ := command.Flags().GetString("database")
 		filestore, _ := command.Flags().GetString("filestore")
 
+		if len(dns) >= 64 {
```

Comment:
We don't normally check stuff in the client and just let the API server validate, but this is fine! Just an FYI.

Reply:
Made here, because if using the API, like me and other robots we can get right away and dont have a two step action.
i can remove if you want 0/5

Reply:
nvm, thought was another file, but if you call using the cmd line it will validate?

Reply:
Yeah, the CLI is just making the same API request which always validates.

Reply:
removed